azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/collection/ResourcesInGroup.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.resources.fluentcore.collection;
+
+public interface ResourcesInGroup<T, DefinitionT> extends
+    SupportsListing<T>,
+    SupportsCreating<DefinitionT>,
+    SupportsDeleting {}
```

Comment:
to clarify, what I mean is 3 interfaces:
`SupportsGetting` with `.get(id)` -- used by non-InGroup
`SupportsGettingByGroup` with `.getByGroup(groupName, resourceName)` -- used by non-InGroup
`SupportsGettingByName` with `.getByName(resourceName)` - which can only be supported by InGroup.